roles/pulp3-postgresql/tasks/main.yml

```diff
@@ -7,16 +7,16 @@
 - block:
 
     # See: https://github.com/ANXS/postgresql/pull/371
```

Comment:
Can you git grep for ANXS and remove the references, I think there are at least 3+